frontend/views/pages/Join.vue

```diff
@@ -75,7 +80,7 @@ export default ({
     pageStatus: {
       get () { return this.ephemeral.pageStatus },
       set (status) {
-        const posibleStatus = ['LOADING', 'WELCOME', 'SIGNING', 'LOGGING', 'INVALID']
+        const posibleStatus = ['LOADING', 'WELCOME', 'SIGNING', 'LOGGING', 'INVALID', 'EXPIRED']
         if (!posibleStatus.includes(status)) {
```

Comment:
Here we have the opportunity to fix that "posibleStatus" -> "possibleStatus" typo.

Reply:
Sure. Will fix soon